sandbox/host/src/sandbox/wasmer_backend.rs

```diff
@@ -116,7 +117,17 @@ pub fn invoke(
     let wasmer_result = SandboxContextStore::using(sandbox_context, || {
         function
             .call(&args)
-            .map_err(|error| Error::Sandbox(error.to_string()))
+            .map_err(|error| {
+                if error.clone().to_trap() == Some(TrapCode::StackOverflow) {
+                    // Panic stops process queue execution in that case.
+                    // This allows to avoid error lead to consensus failures, that must be handled
+                    // in node binaries forever. If this panic occur, then we must increase stack memory size,
+                    // or tune stack limit injection.
+                    // see also https://github.com/wasmerio/wasmer/issues/4181
+                    unreachable!("Suppose that this can not happen, because we have a stack limit instrumentation in programs");
```

Comment:
won't it be the problem of node sync? :/

Reply:
Yes, we should restart webs with this errors

Reply:
Better to know concrete networks that should be restarted

Reply:
Vara Production - OK
VST - OK
VIT - OK
V7 - need to be re started
PST - does not check,  suppose that same as V7

Reply:
PST need to be restarted